src/modules/display.js

```diff
@@ -0,0 +1,93 @@
+import { getTaskListFromLocalStorage, saveTaskListToLocalStorage } from './events.js';
+
+// Function to update the text of a task
+const updateTaskText = (index, newText, isChecked) => {
+  const taskList = getTaskListFromLocalStorage();
+  const taskToEdit = taskList.find((task) => task.index === index);
+
+  if (!taskToEdit) {
+    return;
+  }
+
+  taskToEdit.text = newText;
+  taskToEdit.checked = isChecked || false; // Save the checkbox state in the task object
+  saveTaskListToLocalStorage(taskList);
+};
```

Comment:
- [ ] Please check this function to allow save the edited task to the local storage, it is because I try to edit a task but it didn't actualize the local storage 👇🏽 

![image](https://github.com/Mike47ip/To_Do_List/assets/58642949/45a9b0b8-8833-4214-9e9c-e15dca61ab68)